azavea/ckanext-odp_theme
# Update OIT logo on About page

## Overview

Replaces the logo on the About page (which, unlike the Organizations logos, is a static, tracked file). The old one was only 169 pixels wide while all the others on the page are 200, so this one is 200px wide to match the rest.

The file has already been updated in place on the server.  This is to persist the change for future deployments.

Resolves https://github.com/azavea/opendataphilly-ckan/issues/98

### Demo

Before:
![image](https://user-images.githubusercontent.com/6598836/60737037-3b5cf700-9f27-11e9-82f0-0b96ccc5654a.png)

After:
![image](https://user-images.githubusercontent.com/6598836/60737042-3f891480-9f27-11e9-8422-0c246083a9c0.png)

### Testing instructions

Pull down this branch in your `ckanext-odp_theme` checkout then spin up an ODP instance (see [instructions](https://github.com/azavea/opendataphilly-ckan#development-installation)).  If you already have one, I'm not sure whether you need to reprovision--I think it should work to just start it, since `odp_theme` is symlinked, but my environment had other issues so I had to reprovision anyway.

Go to http://localhost:8025/about.  You should see the new logo.
